client/my-sites/domains/domain-management/transfer/transfer-to-other-user/transfer-domain-to-other-user.jsx

```diff
@@ -257,7 +257,7 @@ class TransferDomainToOtherUser extends Component {
 		const { isMapping, translate, users } = this.props;
 		const availableUsers = this.filterAvailableUsers( users );
 		const saveButtonLabel = isMapping
-			? translate( 'Transfer domain mapping' )
+			? translate( 'Transfer domain connection' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Transfer this domain connection' )` *ES Score: 10*